README.md

```diff
@@ -100,7 +108,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is a collection of logical structures of data, or schema objects. A schema is owned by a database user.
```

Comment:
I really like the first part of your answer; whenever I think of the schema, I think of the structure of a table in a database, what the column headers are, etc.